modules/test/playwright/tests/client-extension-web/editorConfigContributor.spec.ts

```diff
@@ -24,13 +24,12 @@ export const test = mergeTests(
 	editorSamplesPageTest,
 	featureFlagsTest({
 		'LPS-178052': true,
-		'LPS-186870': true,
 	}),
 	isolatedSiteTest,
 	loginTest()
 );
 
-test('Create, edit and delete editor config contributor client extension @LPS-186870', async ({
+test('Create, edit and delete editor config contributor client extension', async ({
```

Comment:
You can leave this in. It's a tag, that matches epic ticket. It just servers for some convenient filtering and searching. [Suggested here](https://liferay.slack.com/archives/C06A59K5D46/p1707314754276979?thread_ts=1707259884.390479&cid=C06A59K5D46).